backends/ebpf/targets/xdp_target.py

```diff
@@ -0,0 +1 @@
+/home/mbudiu/git/p4c/extensions/p4c-xdp/xdp_target.py
```

Comment:
The path in this symlink should be relative.

Reply:
This file should not be here at all, in fact.